config/locales/en.yml

```diff
@@ -23,10 +23,20 @@ en:
       short: "%m/%d"
       time_12_hour: "%l:%M %p %Z"
       day_and_date: "%A, %m/%d"
+  # validation error messages
+  activerecord:
+    errors:
+      models:
+        episode:
+          attributes:
+            segment_count:
+              greater_than: Can't be negative
+              less_than_or_equal_to: Enter a number less than %{count}
```

Comment:
Started translating validation errors.  We'll probably want to audit them all at some point.